cmd/config/internal/commands/cat.go

```diff
@@ -19,15 +21,14 @@ import (
 func GetCatRunner(name string) *CatRunner {
 	r := &CatRunner{}
 	c := &cobra.Command{
-		Use:     "cat DIR...",
+		Use:     "cat DIR",
 		Short:   commands.CatShort,
 		Long:    commands.CatLong,
 		Example: commands.CatExamples,
 		RunE:    r.runE,
+		Args:    cobra.MaximumNArgs(1),
```

Comment:
Should this be exactly one arg?

Reply:
@mortent It should be maximum. As the input can be stdin.